README.md

```diff
@@ -1,17 +1,46 @@
-# smile-identity-core-php
+# Smile Identity PHP Server Side
 
-The Official Smile Identity PHP library
+Smile Identity provides the best solutions for real time Digital KYC, identity verification, user onboarding, and user authentication across Africa. Our server side libraries make it easy to integrate us on the server-side. Since the library is server-side, you will be required to pass the images (if required) to the library.
+If you haven’t already, sign up for a free Smile Identity account, which comes with Sandbox access.
+
+## Dependencies
+
+* Composer build tool
+
+## Documentation
+
+For extensive instructions on usage of the library and sample codes, please refer to the [official Smile Identity documentation](https://docs.smileidentity.com/server-to-server/php)
 
 ## Installation
 
-Download smile-identity-core-php repo to a directory on your server where PHP and Composer is installed.
+### Installing from the Repository
+
+Download [smile-identity-core-php repository](https://github.com/smileidentity/smile-identity-core-php) to a directory on your server where PHP and Composer is installed.
+
+In that directory, run `composer install`
+
+### Installing from Packagist
+
+View the package on [Packagist](https://packagist.org/packages/smile-identity/smile-identity-core).
+
+Alternatively, the package can be searched locally from a composer-based project by typing the command `composer search <PACKAGE_NAME>` in the command line where `PACKAGE_NAME` can the full name of the package (in this case `smile-identity/smile-identity-core`) or any part of the name distinct enough to return a match.
+
+In the project's directory, run `composer install smile-identity/smile-identity-core`
+
+## License
+
+MIT License
+
+## Documentation
+
+For extensive instructions on usage of the library and sample codes, please refer to the official Smile [Identity documentation](https://docs.smileidentity.com/server-to-server/php).
 
-In that directory run `composer install`
+## Getting Help
 
-### Usage
+For usage questions, the best resource is [our official documentation](docs.smileidentity.com). However, if you require further assistance, you can file a [support ticket via our portal](https://portal.smileidentity.com/partner/support/tickets) or visit the [contact us page](https://portal.smileidentity.com/partner/support/tickets) on our website.
 
-Edit the example_core.php file and replace sections marked with <>
+## Contributing
 
-### Run tests
+Bug reports and pull requests are welcome on GitHub at https://github.com/smileidentity/smile-identity-core-php
 
-In same directory run `vendor/bin/phpunit tests`
+Please format the code with [black](https://github.com/psf/black) prior to submitting pull requests, by running `black .` from the project's root.
```

Comment:
I didn't realize black also formats PHP code

Reply:
Don't be smart with me now @macphilips 😂😂😂
Unfortunate copy-paste! 🙏🏽